src/Tbody.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import fakeBookings from "./fakeBookings.js";
```

Comment:
Ok, I will let this question for you. Should this component import `fakeBookings` directly, o should receive it via props?

Reply:
I expect it to be imported directly, in fact i had both versions, first i was sending it from another component and in the end i thought that just importing it is safer and all was still working  and i did not mind to check the logic from the beginning to end

Reply:
at this moment i prefer  original unchanged data, but on later steps if i will be filtering the array before creating cards it's good to have it not imported directly but stored in variable